src/components/task/crumb-mixin.cjsx

```diff
@@ -57,7 +57,39 @@ module.exports =
   _generateCrumbs: (id) ->
     task = TaskStore.get(id)
     steps = TaskStore.getSteps(id)
-    @_generateCrumbsFromSteps(task, steps)
+    crumbs = @_generateCrumbsFromSteps(task, steps)
+    @modifyCrumbs(task, crumbs)
+    crumbs
+
+
+  # can possibly abstract this more and pull this out somewhere else if needed in the future
+  modifyCrumbs: (task, crumbs) ->
+    {currentStep} = @props
+
+    # insert spacer panel/crumb for reading task that have spaced practices or personalized problems
+    if task.type is 'reading'
+      notCore = _.find crumbs, (crumb) ->
+        (crumb.type is 'step') and not TaskStepStore.isCore(crumb.data.id)
+
+      if notCore?
```

Comment:
oh wait, that line determines if there is at least one non-core question found, so if `notCore?` is true, that means there is at least one question after the last core question